docs/parameter_tuning.md

```diff
@@ -28,6 +28,24 @@ determine which ones to tweak to obtain the desired output.
   - [Limited Computing Resources](#limited-computing-resources)
 - [Output Units](#output-units)
 
+## Interactive parameter explorer
+
+To build intuition about what some CNMF-E parameters do on real datasets, 
+the following interactive tools provide instantaneous feedback showing
+you what effect these parameters:
+
+- the cell diameter
```

Comment:
we can review/update the IDPS user guide if this is not explained well, but the recommendation should be to set the cell diameter to the average cell diameter of a representative cell (this can be measured using the line measurement tool in IDPS) and then play with the other parameters to optimize cell identification

Reply:
done! added a sentence to the widget saying that users should fix the size and the mess with other params.